src/main/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseController.java

```diff
@@ -147,15 +147,32 @@ HearingNoteResponse createHearingNote(@PathVariable(value = "hearingId") String
                                           @Valid @RequestBody HearingNoteRequest hearingNoteRequest,
                                           Principal principal) {
 
-        if (!StringUtils.equals(hearingId, hearingNoteRequest.getHearingId())) {
-            throw new ConflictingInputException(String.format("Hearing Id '%s' provided in the path does not match the one in the hearing note request body submitted '%s'",
-                    hearingId, hearingNoteRequest.getHearingId()));
-        }
+        validateHearingNoteRequest(hearingId, hearingNoteRequest.getHearingId());
 
         HearingNoteEntity hearingNote = hearingNotesService.createHearingNote(hearingNoteRequest.asEntity(authenticationHelper.getAuthUserUuid(principal)));
         return HearingNoteResponse.of(hearingNote);
     }
 
+    private static void validateHearingNoteRequest(String hearingId, String hearingNoteRequest) {
```

Comment:
Please rename the second parameter to like hearingIdFromNoteRequest.

Reply:
Addressed